ceph/rados/core_workflows.py

```diff
@@ -4615,3 +4615,68 @@ def get_rados_df(self, pool_name: str = None):
         out = self.run_ceph_command(cmd=_cmd, client_exec=True)
 
         return out["pools"][0] if pool_name else out
+
+    def set_configuration(self, who: str, param: str, value: str) -> int:
+        """
+        Module to set the configuration parameter of daemons
+        Args:
+            who: global or daemon ( osd, on, mgr, mds )
+            params : Parameters to be set
+
+        Returns:
+            success -> True
+            failure -> False
+
+        Usage:
+            rados_obj.set_configuration(who="osd", param="osd_mclock_override_recovery_settings", value="true")
+            rados_obj.set_configuration(who="mgr", param="mgr/dashboard/ceph1/server_addr", value="10.74.250.90")
+        """
+        log.info(f"Setting configuration parameter {param} on {who}")
+        cmd = f"ceph config set {who} {param} {value}"
+        _ = self.run_ceph_command(cmd=cmd)
+        actual_value = self.get_configuration(who=who, param=param)
+        log.info(f"actual value: {actual_value}")
+        log.info(f"value: {value}")
+        if actual_value == value:
+            log.info(f"Configuration parameter {param} is set with {value} on {who}")
+            return True
+        else:
+            return False
+
+    def get_configuration(self, who: str, param: str) -> int:
```

Comment:
Module for this already exists. please refer to `get_config()` in `tests/rados/monitor_configurations.py`

Reply:
Updated code to reuse function get_config() from tests/rados/monitor_configurations.py

https://github.com/red-hat-storage/cephci/blob/262205193a7497c6e25014f51a479518e7a1ba85/tests/rados/test_bz_2213766.py#L85-L88